admin/src/css/theme.less

```diff
@@ -1,25 +1,47 @@
-.body,
-.header {
-  padding: 20px;
+@nav-active-color: #38464d;
+body {
+  max-width: @admin-content-width;
+  margin: 0 auto;
+  font-size: @font-medium;
 }
 .body {
   display: flex;
 }
+.body>.content {
+	margin: 30px 0 0 30px;
```

Comment:
Indentation is off - make sure your editor is configured to use 2 spaces not 1 tab for indentation.